main.py

```diff
@@ -2,6 +2,7 @@
 
 from PyQt6 import uic
 from PyQt6.QtWidgets import *
+from PyQt6.QtCore import *
```

Comment:
does this need to be added to the requirements documentation?

Reply:
Should be fine I believe